wiki/Ranking_Criteria/Difficulty_Naming/en.md

```diff
@@ -0,0 +1,139 @@
+# Difficulty Naming
+
+This wiki article serves as a substitute and an update to [this topic](https://osu.ppy.sh/community/forums/topics/178700). 
```

Comment:
linking old topic undermines purpose really hard lmao

Reply:
nah, the topic is the direct reference i took for this article, not showing the source looks a bit rude because not giving proper credit to the original thread creator. it's already explicitly stated that this is the **updated version** so people should focus on here but OH weLL

Reply:
pm'd priti instead to ask him if he cares /shrug